### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by jasonkhadka**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/issue------feature-request.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
```
Comment (sjrl) [inline .github/ISSUE_TEMPLATE/issue------feature-request.md:1]: Quick clarification. Why do we need this file `issue------feature-request.md` and this file `feature_request.md`?

Comment (jasonkhadka) [reply .github/ISSUE_TEMPLATE/issue------feature-request.md:1]: yeah quite strange, these are auto generated files by github. probably something to do with how these templates work.

Comment (sjrl) [reply .github/ISSUE_TEMPLATE/issue------feature-request.md:1]: yeah weird, I also don't see this files in pytorch's repo for example https://github.com/pytorch/pytorch/tree/master/.github/ISSUE_TEMPLATE

**Comment by jasonkhadka** [comment]
deleted extra files.

File: .github/ISSUE_TEMPLATE/documentation.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+name: "Issue: \U0001F4DC Documentation "
+about: Describe this issue template's purpose here.
```
Comment (sjrl) [inline .github/ISSUE_TEMPLATE/documentation.md:3]: Replace the `about:` boiler plate with something like `Report an issue related to https://pytorch.org/docs`

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+---
+name: "Issue: 🛰️ Feature request"
+about: Suggest an idea for this project
```
Comment (sjrl) [inline .github/ISSUE_TEMPLATE/feature_request.md:3]: Rewrite to `Submit a proposal/request for a new PADL feature`

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
+---
+name: "Issue: \U0001F41E Bug report"
+about: Create a report to help us improve
```
Comment (sjrl) [inline .github/ISSUE_TEMPLATE/bug_report.md:3]: `us improve PADL`

**Review by wuhu — APPROVED** [review]

**Pull request merged**